src/test/java/core/basesyntax/service/CsvFileReaderServiceImplTest.java

```diff
@@ -12,6 +12,10 @@
 
 class CsvFileReaderServiceImplTest {
     private static CsvFileReaderService fileReaderService;
+    private String correctFile = "src/main/java/core/basesyntax/resources/Correct.csv";
```

Comment:
```suggestion
    private static final String CORRECT_FILE_PATH = "src/main/java/core/basesyntax/resources/Correct.csv";
```
please correct all the constants)